SaintCoinach/ex.json

```diff
@@ -11609,6 +11698,14 @@
         {
           "index": 9,
           "name": "HuntingLogExpReward"
+        },
+        {
+          "index": 11,
+          "name": "ItemLevelSync"
+        },
+        {
+          "index": 14,
+          "name": "cLevel"
```

Comment:
```suggestion
          "name": "ClassJobLevel"
```

Is this "ClassJob" ?

Reply:
supposedly crafterlevel?, apparently a level for crafting that is used to determine potency penalties/bonuses

Reply:
Maybe CraftingLevel? C is too ambiguous.